utils/api/configurations.js

```diff
@@ -185,37 +185,45 @@ export const  configureFiles = (data) => {
   return allFiles
 }
 
-export const configureDocuments = (documents, requestIdentifier) => {
-  return documents?.map(document => ({
-    identifier: document.identifier,
+export const configureDocument = (document, requestIdentifier) => {
+  return {
     date: normalizeDate(document.created_at),
     documentStatus: document.status,
     documentStatusColor: statusColors[configureStatus(document.status)].bg,
     documentType: document.type,
     documentTypeColor: 'bg-dark',
+    identifier: document.identifier,
     lineItems: configureLineItems(document.line_items),
     requestIdentifier,
-    subtotalPrice: document.retail_subtotal_price_currency,
-    taxAmount: document.tax_cost_currency,
-    terms: document.payment_terms,
-    totalPrice: document.retail_total_price_currency,
     shippingPrice: document.shipping_cost_currency,
     shipTo: {
-      organizationName: document.ship_to?.organization_name,
-      text: document.ship_to?.text,
+      organizationName: document.ship_to.organization_name,
+      text: document.ship_to.text,
     },
     shipFrom: {
-      organizationName: document.ship_from?.organization_name,
-      text: document.ship_from?.text,
+      organizationName: document.ship_from.organization_name,
+      text: document.ship_from.text,
     },
-    // the following properties only need to exist on POs
-    turnaroundTime: document.turn_around_time.human || null,
-    poNumber: document.po_number || null,
-    relatedSOWIdentifier: document.proposal_ref?.identifier || null,
-    adPO: document.scientist_identifier || null,
+    subtotalPrice: document.retail_subtotal_price_currency,
+    taxAmount: document.tax_cost_currency,
+    terms: document.payment_terms,
+    totalPrice: document.retail_total_price_currency,
+  }
+}
+
+export const configureSOWs = (sows, requestIdentifier) => {
+  return sows?.map((sow) => ({
+    ...configureDocument(sow, requestIdentifier),
```

Comment:
although the sow is the same as the document at this time, this distinction will make it easier for us going forward as we add more document types.

and in the event the sow properties change, we can change them here.